src/Mvc/Mvc.Cors/src/CorsAuthorizationFilter.cs

```diff
@@ -110,8 +110,7 @@ public async Task OnAuthorizationAsync(Filters.AuthorizationFilterContext contex
                     !StringValues.IsNullOrEmpty(accessControlRequestMethod))
                 {
                     // If this was a preflight, there is no need to run anything else.
-                    // Also the response is always 200 so that anyone after mvc can handle the pre flight request.
-                    context.Result = new StatusCodeResult(StatusCodes.Status200OK);
+                    context.Result = new StatusCodeResult(StatusCodes.Status204NoContent);
```

Comment:
This matches what the CORS middleware does.

Reply:
I'm not too concerned about changing 200 to 204, consistency is good.

Reply:
I looked and the spec says any 2XX is legal.